lib/oban/job.ex

```diff
@@ -201,6 +201,7 @@ defmodule Oban.Job do
 
   @unique_fields ~w(args meta queue worker)a
   @unique_timestamps ~w(inserted_at scheduled_at)a
+  @key_compatible_fields ~w(args meta)a
```

Comment:
Shortening the attribute name for conciseness below. Also, so that @sorentwo gives me a smooch.

```suggestion
  @keyable_fields ~w(args meta)a
```